automation_testing/tests/Components/dataview/save_as.spec.ts

```diff
@@ -91,11 +91,10 @@ test.describe.parallel("Components - Data View - Save As", () => {
 		await saveAs.viewBtn.click();
 		await saveAs.wait();
 		const editBtn = await saveAs.editBtnByLabel(saveAs_data.saveAsView);
-		// await saveAs.validateSnapshot(editBtn, "save_as_edit_view_btn");
 		expect(await (await saveAs.getViewTypeByLabel(saveAs_data.saveAsView)).textContent()).toContain(saveAs_data.viewNotSharedType);
 		await editBtn.click();
 		await saveAs.saveLabel.fill(saveAs_data.saveAsViewEdit);
-		expect((await saveAs.editCheckbox.isChecked()).valueOf()).toBe(false);
+		await expect(saveAs.saveAsCheckbox).not.toBeChecked();
```

Comment:
I'm getting the following error in this line: 
![image](https://user-images.githubusercontent.com/23409284/200419320-9b47da99-0790-4cf0-8750-6e4e38e130ef.png)

Reply:
Fixed